README.md

```diff
@@ -13,7 +13,22 @@ This service is designed as part of the SDP and other services are available thr
 
 The SDP Vocabulary Service requires: Ruby (version 2.3 or later), bundler (version 1.13.6 or later), Node.js (version 5.5 or later) and Postgres (version 9.6 or later).
 
-### Install Dependencies
+### Install Elasticsearch
+
+The service relies on an Elasticsearch index for searching content in the application.
+Information pertaining to downloading and installtion Elasticsearch can be found [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/_installation.html)
+
+By default, the Vocabulary Service expects the Elasticsearch server to be running locally on the same system and accessible  via 127.0.0.1 on port 9200.  This can be configured in the config/settings.yml file by providing host and port parameters in the elasticsearch section of the configuration.   
+
+Please note that the config/settings.yml file works inconjuntion with the files in the config/settings directory.  Each file in said directory provides overrides for any given Rails environment that the service will be run in.  In addition, another settings file, config/settings.local.yml, which is not checked into the repository, provides local developer overrides for all environments.  So the hierarchy of settings files goes config/settings.yml -> config/settings/(environment).yml -> config/settings.local.yml
+
+### Install Concept Service
+
+While not absolutely critical for the Vocabulary Service to function, the Concept Service provides users the ability to look up codes in different code systems for different tasks inside the application.  These tasks can also be preformed by manual entry so the lookup service is a convenience function for a better end user experience.  Installation of the Concept Service can be found in [here](https://github.com/CDCgov/concept-dictionary-manager)
```

Comment:
What happens if the concept service is not installed?

"Installation instructions for the Concept service can be found here"